ui/src/message_popup/author/question_page.jsx

```diff
@@ -104,8 +106,11 @@ export default React.createClass({
     console.log("Saving the following new question:");
     const studentIds = this.state.students.map(student => student.id);
     const text = this.state.questionText;
+    const {originalQuestion} = this.props;
     const {goodExamples, badExamples} = this.getExamples();
-    const id = _.maxBy(allQuestions, question => question.id).id + 1;
```

Comment:
Let's talk out in person how this component could require a prop like `onQuestionEdited`.  If we did that, I think we could remove all this code and the need for this component (which is about editing a single question) to have to know about all the questions in the database.  The component above could handle that.

Similar for `onCreate` and `onDelete`.